### ISSUE TITLE
1.7.2 bug

### ISSUE THREAD

**Issue by AntoineD** [PyCQA/docformatter#234]
With
```python
def f():
    """CC.

    :math:`f(0) = 1`. XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX XXXXXXXXXXXXXXXXXXXXXX
    """
```

which produces inserts a blank space after the second `:`
```python
def f():
    """CC.

    :math: `f(0) = 1`. XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
        XXXXXXXXXXXXXXXXXXXXXX
    """
```

**Comment by AntoineD** [comment]
I'm on linux with python 3.10.11 if that matters and
```console
❯ pip list        
Package            Version
------------------ -------
charset-normalizer 3.1.0
docformatter       1.7.2
pip                23.1.2
setuptools         67.7.2
untokenize         0.1.1
wheel              0.40.0
```

**Comment by weibullguy** [comment]
Reproduced and found the fix.

**Comment by weibullguy** [comment]
@AntoineD this should be fixed in v1.7.3-rc1
